compiler/utils.go

```diff
@@ -643,3 +644,38 @@ func endsWithReturn(stmts []ast.Stmt) bool {
 func encodeIdent(name string) string {
 	return strings.Replace(url.QueryEscape(name), "%", "$", -1)
 }
+
+// jsTagFormat returns a string of Javascript code appropriate for accessing
+// the property identified by jsTag. If the jsTag value can be safely encoded
+// in Javascript using the dot notation this is used, else we fallback to the
+// bracket notation. For more details see:
+//
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_Accessors
+//
+// Uses definition of an identifier from
+// https://developer.mozilla.org/en-US/docs/Glossary/Identifier
+func jsTagFormat(jsTag string) string {
```

Comment:
This is entirely a nitpick, so ignore it if you wish... But I wonder if `formatJSTag` would be a better name, and more consistent with the other func names here, which use the `<verb><object>` convention (i.e. `formatExpr` and `encodeIdent` etc)

Reply:
Done.